update-dois/src/main/java/org/reactome/release/updateDOIs/Main.java

```diff
@@ -47,8 +47,7 @@ public static void main( String[] args ) throws IOException {
       String hostGK = props.getProperty("curator.database.host");
       String databaseTR = props.getProperty("release_current.name");
       String databaseGK = props.getProperty("curator.database.name");
-      authorIdTR = Integer.valueOf(props.getProperty("personId"));
-      authorIdGK = Integer.valueOf(props.getProperty("personId"));
+      personId = Integer.valueOf(props.getProperty("personId"));
```

Comment:
The `personId` variable can be declared here instead of line 33 once the `try` block is removed.  

It also may be better parsed using `Long.parseLong(props.getProperty("personId"))` so a `long` value is returned rather than an `int` (though, Java will cast for you implicitly).